api/control/v1alpha1/types.go

```diff
@@ -394,27 +394,14 @@ const (
 	AggregatorTypeMax   AggregatorType = "MAX"
 )
 
-// Expression is some value to derive from a workload object from a WEC.
-// An Expression is either a JSONPath identifying a value to extract
-// or a boolean combination of comparisons of atomic values.
-// While the expression here in the Go type system admits other values,
-// the controller will accept only the restricted set stated above.
-// Post-apiserver validation errors are posted to the StatusCollectorStatus.
-type Expression struct {
-	Op   ExpressionOperator `json:"op"`
-	Path string             `json:"path,omitempty"`
-	Args []Expression       `json:"args,omitempty"`
-}
-
-type ExpressionOperator string
-
-const (
-	OperatorPath  ExpressionOperator = "Path" // JSONPath string
-	OperatorOr    ExpressionOperator = "Or"
-	OperatorAnd   ExpressionOperator = "And"
-	OperatorNot   ExpressionOperator = "Not"
-	OperatorEqual ExpressionOperator = "Equal"
-)
+// Expression is written in the Common Expression Language.
+// See github.com/google/cel-go/cel for the Go implementatiom used in Kubernetes.
```

Comment:
`github.com/google/cel-go/cel` does not exist.

Reply:
Sorry, I meant github.com/google/cel-go

Reply:
The force-push to 46fc894fa fixes this typo.